cluster-up/cluster/k8s-provider-common.sh

```diff
@@ -14,7 +14,16 @@ function deploy_kwok() {
     fi
 }
 
-
+# copy_istio_cni_conf_files copy the generated Istio CNI net conf file
+# (at '/etc/cni/multus/net.d/') to where Multus expect CNI net conf files ('/etc/cni/net.d/')
+function copy_istio_cni_conf_files() {
```

Comment:
@aerosouund Is this something new or did we have this before?

Reply:
it was there before